hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/execution/HederaEvmTxProcessor.java

```diff
@@ -168,6 +168,8 @@ public HederaEvmTransactionProcessingResult execute(
         this.gasUsed = calculateGasUsedByTX(gasLimit, initialFrame);
         this.sbhRefund = updater.getSbhRefund();
 
+        tracer.finalizeOperation(initialFrame);
+
```

Comment:
Actually, we should have this called in `EvmTxProcessor`, since the `process` call at line 165 may throw a `ResourceLimitException` exception, which is [caught in EvmTxProcessor](https://github.com/hashgraph/hedera-services/blob/cdcfb8885cef3dc43d3a919b2ef120807ae400ee/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/execution/EvmTxProcessor.java#L138-L140).
If thrown, this call here will be skipped.